<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>复杂网络视频图像文字提取与关键词匹配系统</string>
  </property>
  <widget class="QPushButton" name="pushButton_run">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>130</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_det">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CTPN</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;检测模型&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_rec">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SPDN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ISMN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CRNN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Attention</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;识别模型&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>690</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;匹配算法&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_mat">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AC自动机</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>模糊匹配</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="radioButton_mat">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>是否执行关键词匹配</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_recResult">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>740</y>
     <width>991</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_detResult">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>991</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string>检测结果</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_init">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>130</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_uploaddir">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_uploadfile">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
